samples/javascript/02.react-video/setup.js

```diff
@@ -11,7 +11,7 @@ module.exports = async function () {
     console.log(chalk.green("Setup Puppeteer"));
     const browser = await puppeteer.launch({
         headless: false,
-        executablePath: "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
+        executablePath: "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe",
```

Comment:
this is OS specific, can you please add a comment with what Mac users need to change it to?